### DISCUSSION TITLE
I have a few questions that I want to consult with the professionals of Kyuubi community. Can you help me answer them? thank

### DISCUSSION THREAD

**Discussion by 2416210017** [apache/kyuubi#4263] [Q&A]
1. When using hue+kyuubi, we found that when selecting a database with kudu table, hue will directly report an error and the table name will not be displayed

![image](https://user-images.githubusercontent.com/52597892/217215425-0f240120-63df-4c4f-9b26-65a327128d6f.png)
![image](https://user-images.githubusercontent.com/52597892/217215556-03d6ab61-56dc-4efb-85c3-a1f80e8d1eeb.png)

2. The default is hue+kyuubi, and the spark engine is used. Can hue be configured to support the flink engine?

3. Kyuubi supports the hive engine, but only the hive 3.1 version, but we use the hive 2.2.1 version, which supports the lower version of hive. Will this increase in the future?

**Comment by pan3793** [comment]
1. The direct cause is ClassNotFound as the first picture shows. But I'm not sure if it works even if you put the Kudu-related jar under `SPARK_HOME/jars`.

2. Technically it could, please refer to Kyuubi docs to set up and tune `kyuubi-defaults.conf`

3. It was listed but not scheduled, it's an open-source project, if it's your case, free feel to send PR, Kyuubi community welcomes new contributors, any time.

Tips: Please paste the text instead of the picture when sharing stacktrace/logs.
